benchmark/browser/scenarios/sx-prop-box-theme-ui/index.js

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+/** @jsx jsx */
+import {
+    jsx,
+    Box,
+    ThemeProvider
+  } from 'theme-ui';
+import { alpha } from '@material-ui/core/styles';
+import { logReactMetrics } from '../utils';
+
+const theme = {
+  fonts: {
+    body: 'system-ui, sans-serif',
+    heading: '"Avenir Next", sans-serif',
+    monospace: 'Menlo, monospace',
+  },
+  colors: {
+    text: '#000',
+    background: '#fff',
+    primary: '#33e',
+  },
+}
+
+export default function BasicStyledComponents() {
+    return (
+      <React.Profiler id="sx-prop-box-theme-ui" onRender={logReactMetrics}>
+        {new Array(1000).fill().map(() => (
+          <ThemeProvider theme={theme}>
+            <Box
```

Comment:
Shouldn't we use a `div` here? It seems that in theme-ui, you either pick a Box or the jsx pragma.

Reply:
I choose the Box because I am using the Box from @material-ui too. I will add another one with div, just to see if it will make big difference

Reply:
I'm asking because the sx feature of theme-ui doesn't come from the Box component.

Reply:
I've added example for both div and the Box component for comparison